service/src/main/resources/dev-data/espoo-dev-data.sql

```diff
@@ -1928,6 +1928,16 @@ INSERT INTO service_need_option (name, valid_placement_type, default_option, fee
     ('Osaviikkoinen, enintään 25h', 'DAYCARE', FALSE, 0.6, 0.6, 1.0, 25, FALSE, TRUE),
     ('Osapäiväinen', 'DAYCARE_PART_TIME', FALSE, 0.6, 0.6, 0.54, 25, TRUE, FALSE),
     ('Osapäiväinen ja osaviikkoinen', 'DAYCARE_PART_TIME', FALSE, 0.6, 0.6, 0.54, 25, TRUE, TRUE),
+    ('5-vuotiaiden kokopäiväinen, vähintään 45h', 'DAYCARE_FIVE_YEAR_OLDS', FALSE, 0.8, 1.0, 1.0, 45, FALSE, FALSE),
+    ('5-vuotiaiden kokopäiväinen, 35-45h', 'DAYCARE_FIVE_YEAR_OLDS', FALSE, 0.6, 1.0, 1.0, 40, FALSE, FALSE),
+    ('5-vuotiaiden kokopäiväinen, 25-35h', 'DAYCARE_FIVE_YEAR_OLDS', FALSE, 0.35, 1.0, 1.0, 35, FALSE, FALSE),
+    ('5-vuotiaiden osaviikkoinen, 25-35h', 'DAYCARE_FIVE_YEAR_OLDS', FALSE, 0.35, 1.0, 1.0, 35, FALSE, TRUE),
+    ('5-vuotiaiden osaviikkoinen, 20-25h', 'DAYCARE_FIVE_YEAR_OLDS', FALSE, 0.35, 0.6, 1.0, 25, FALSE, TRUE),
+    ('5-vuotiaiden ilmainen osaviikkoinen', 'DAYCARE_FIVE_YEAR_OLDS', FALSE, 0.0, 0.6, 1.0, 20, FALSE, TRUE),
+    ('5-vuotiaiden osapäiväinen, 20-25h', 'DAYCARE_PART_TIME_FIVE_YEAR_OLDS', FALSE, 0.35, 0.6, 0.5, 25, TRUE, FALSE),
+    ('5-vuotiaiden osapäiväinen, enintään 20h', 'DAYCARE_PART_TIME_FIVE_YEAR_OLDS', FALSE, 0.0, 0.6, 0.5, 20, TRUE, FALSE),
+    ('5-vuotiaiden osapäiväinen ja osaviikkoinen, 20-25h', 'DAYCARE_PART_TIME_FIVE_YEAR_OLDS', FALSE, 0.35, 0.6, 0.5, 25, TRUE, TRUE),
+    ('5-vuotiaiden ilmainen osapäiväinen ja osaviikkoinen', 'DAYCARE_PART_TIME_FIVE_YEAR_OLDS', FALSE, 0.0, 0.6, 0.5, 20, TRUE, TRUE),
```

Comment:
maybe we could make an excel sheet out of these at some point and let customer validate that all coefficients etc are correct?

Reply:
Yeah, we probably should.